tensorflow/python/keras/losses_test.py

```diff
@@ -1578,6 +1578,18 @@ def test_non_default_delta(self):
     actual_loss = sample_weight * np.sum(self.expected_losses) / self.batch_size
     self.assertAlmostEqual(self.evaluate(loss), actual_loss, 3)
 
+  def test_all_correct(self):
```

Comment:
Let's rename the test name since "all_correct" seems to be too generic. How about test_loss_with_non_default_dtype?

Reply:
@qlzh727 Thanks. Updated.